java/vector/src/test/java/org/apache/arrow/vector/TestValueVector.java

```diff
@@ -23,16 +23,12 @@
 
 import java.nio.charset.Charset;
 
+import org.apache.arrow.memory.BufferAllocator;
 import org.apache.arrow.memory.RootAllocator;
 import org.apache.arrow.vector.complex.ListVector;
 import org.apache.arrow.vector.complex.MapVector;
 import org.apache.arrow.vector.complex.RepeatedListVector;
 import org.apache.arrow.vector.complex.RepeatedMapVector;
-import org.apache.arrow.vector.types.MaterializedField;
-import org.apache.arrow.vector.types.Types;
-import org.apache.arrow.vector.types.Types.MinorType;
-import org.apache.arrow.vector.util.BasicTypeHelper;
-import org.apache.arrow.vector.util.OversizedAllocationException;
 import org.apache.arrow.vector.holders.BitHolder;
```

Comment:
yes, I can add checkstyle plugin for import order. Any preference? http://checkstyle.sourceforge.net/config_imports.html#CustomImportOrder_Example (I'm using the Eclipse order, and I forgot to disable "organize imports on save action")